enhancements/image-references.md

```diff
@@ -0,0 +1,225 @@
+ecosystem, which is directly opposed to the goal of this enhancement.
+
+## Design Details
+
+The definition of a minimally-viable operator bundle image will be changed to
+ensure that all iamge references in the contained manifests have been resolved 
+to a digest and had the pull specifications rewritten to refer to those digests.
+
+The only acceptable process for creating an operator bundle will be to run the
+`operator-sdk bundle create` CLI, providing the manifests, metadata and image
+sources as input to the creation process.
+
+### Proposed UX
+
+Operator manifest authors write a manifest that refers to images using some
+opaque string [ex](https://github.com/openshift/machine-config-operator/blob/master/install/0000_80_machine-config-operator_04_deployment.yaml#L22),
+and provide an `image-references` file alongside their manifests that declares
+which strings inside of their manifest are referring to pull specifications of
+images and names each occurence [ex](https://github.com/openshift/machine-config-operator/blob/98d9ba6841eb4811ed6f4d4de7016ea83c131c54/install/image-references#L7-L10).
+
+The `image-references` file holds data in the format of an OpenShift `ImageStream`:
+
+```yaml
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+spec:
+  tags:
+  - name: my-image-name # this is the semantic identifier for this image
+    from:
+      kind: DockerImage
+      name: registry.svc.ci.openshift.org/openshift:image # this is an opaque string which exists in manifests
+```
+
+This mapping, therefore, defines what needs to be replced in manifests when
+they are bundled and identifies each replacement with a name. When
+`operator-sdk bundle create` runs, it will require as input a second mapping
+from those names to literal image pull specifications and will run the replacement.
+In this manner, the configuration provided by the manifest author remains static
+regardless of the eventual replacement that a build system will execute.
+
+The second mapping required at build-time will also take the form of an `ImageStream`
+and will be provided either via reading an `ImageStream` from an OpenShift cluster or
+reading a serialized `ImageStream` from disk. A build system that does not store the
+images used as sources in an `ImageStream` may therefore provide a static file instead.
+The mapping to provide the image sources looks like:
+
+```yaml
+apiVersion: image.openshift.io/v1
+kind: ImageStreamImageam
+status:
+  tags:
+  - tag: my-image-name # this matches the semantic identifier provided by the manifest author
+    items:
+    - dockerImageReference: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:298d0b496f67a0ec97a37c6e930d76db9ae69ee6838570b0cd0c688f07f53780 # this is the fully-resolved pull specification that will be used in replacements
+      image: sha256:298d0b496f67a0ec97a37c6e930d76db9ae69ee6838570b0cd0c688f07f53780
+```
+
+`operator-sdk bundle create` will use this chain of mapping to perform replacements
```

Comment:
It may be worth calling out that this image is not needed to be multi-arch. As well that the image will have a `media-type: application/vnd.docker.distribution.manifest.v2+json` and be conforming to the version 2 schema 2 spec?